tests/spelling.R

```diff
@@ -1 +1,4 @@
-spelling::spell_check_test(vignettes = TRUE, error = identical(Sys.getenv("TRAVIS"), "true"))
+if (requireNamespace("spelling")) {
+    spelling::spell_check_test(vignettes = TRUE, error = identical(Sys.getenv("TRAVIS"), "true"))
```

Comment:
¿can this just go away if we don’t care about travis builds at all anymore?

Reply:
Oh, missed that. I think it would be best to create a github_action env var and error based on that. This would match what rcrunch does. (and I'm a terrible speller, so I kind of need the check)